var/spack/repos/builtin/packages/netcdf-cxx4/package.py

```diff
@@ -36,9 +36,6 @@ class NetcdfCxx4(Package):
     depends_on('netcdf')
     depends_on("autoconf", type='build')
 
-    def install(self, spec, prefix):
+    def autoreconf(self, spec, prefix):
```

Comment:
@alalazo 
Is there a more "clever" way in AutotoolsPackage to just say "this needs autoreconf"?